courses/free-courses-ru.md

```diff
@@ -65,6 +65,7 @@ ADV - Продвинутый. Тонкости.
 ### Elixir
 
 * [Язык программирования Эликсир](https://github.com/yzh44yzh/elixir_course) - Yuri Zhloba
+* [Elixir - функциональная разработка](https://www.youtube.com/watch?v=lZtdNCkevVw&list=PLWlFXymvoaJ_SWXOOm2JSqv86ZBkQ9-zo) - Ilya Krukowski (BEG)
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

Reply:
Changed the link to the playlist. The author is also indicated (Ilya Krukowski).